.storybook/components/PicassoBook/Chapter.tsx

```
@@ -114,14 +114,14 @@ class Chapter extends Base {
     }
 
     if (!component.__docgenInfo) {
-      throw new Error(
+      console.error(
```

I think we should throw an error here, so the build will fail on CI

---

good point, it didn't come to my mind, that error would trigger a fail on CI. I will revert it.